hooks/use-3d-card-effect.ts

```
@@ -33,66 +33,61 @@ export function use3DCardEffect({
+
+        setTransform(`perspective(1000px) scale(${settings.scale}) rotateX(${rotateX}deg) rotateY(${rotateY}deg)`)
+        setGlowPosition({
+          x: ((e.clientX - rect.left) / rect.width) * 100,
+          y: ((e.clientY - rect.top) / rect.height) * 100,
+        })
+      })
+    },
+    [settings.tilt, settings.scale]
+  )
+
   useEffect(() => {
```

Biome (2.4.4)

[error] This hook is being called from within a function or method that is not a hook or component.

(lint/correctness/useHookAtTopLevel)